tony-history-server/test/controllers/JobsConfigPageControllerTest.java

```diff
@@ -0,0 +1,24 @@
+package controllers;
+
+import org.junit.Test;
+import play.Application;
+import play.inject.guice.GuiceApplicationBuilder;
+import play.test.WithApplication;
+
+import static org.junit.Assert.*;
+import static play.mvc.Http.Status.*;
+
+
+public class JobsConfigPageControllerTest extends WithApplication {
```

Comment:
Is this the replacement for the HomeControllerTest? If so, we should migrate the tests over.

Reply:
No its not. This one requires access to HDFS to read file, so I will look into how to create mock HDFS obj in Play.